### ISSUE TITLE
Error at sign in BC Container: "You cannot sign in due to a technical issue"

### ISSUE THREAD

**Issue by salmosa1** [microsoft/navcontainerhelper#1308]
I've created a new container using the new module BCContainerHelper. The container has been create with the next script,

```
$artifactUrl = Get-BCArtifactUrl -version 16 -country es -select Latest
$credential = New-Object pscredential 'UserName', (ConvertTo-SecureString -String 'Password' -AsPlainText -Force)

New-BCContainer -accept_eula `
    -alwaysPull `
    -artifactUrl $artifactUrl `
    -containerName "Name" `
    -auth UserPassword `
    -credential $credential `
    -licenseFile "LicensePath" `
    -includeAL `
    -updateHosts `
    -assignPremiumPlan `
    -accept_outdated

```
PowerShell creates it without any issue, but when I try to sign in, BC throws the error "You cannot sign in due to a technical issue"

BCContainerHelper is version 1.0.6
BCContainerHelper is running as administrator
Host is Microsoft Windows 10 Pro (Last Version)
Docker Version is 19.03.12

I've tried to create a container of BC 17 and it throws the same error.

What I'm doing wrong?

Thanks!

**Comment by freddydk** [comment]
I assume that you are using sandbox artifacts (as you mention 17) - the template says you should include full output from the container generation as well.
Also, I have formattet the code above so that it is readable (use the Insert Code)

I assume that you need to specify ?tenant=default in the URL (that should be in the Web Client shortcut)

**Comment by salmosa1** [comment]
Sorry, I follow the template, but I didn't know about the quotes to insert the code. I wrote the lines with the artifact too, with the version I tried (16). About the output I didn't have the output, but, as I said, there was nothing strange or different with others containers I made with the navcontainerhelper, so I assume that if all is normal, it wasn't needed.

Otherwise, I've tried what you said, and as I thought, I was doing something wrong ;) I never specify this in any docker, (?tenant=default), and I thought it wasn't mandatory now.

Thanks for your fast answer!
